src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+
+import Icon from "../Icon/Icon";
+import {
+  Selector,
+  Item,
+  Wrapper,
+  Arrow,
+  AbsoluteWrapper,
+} from "./Dropdown.styles";
+
+export type SelectOption = {
+  label: string;
+  value: string;
+};
+
+export type DropdownProps = {
+  category: SelectOption;
+  options: SelectOption[];
+  onChange: (option: SelectOption) => void;
+};
```

Comment:
```suggestion
  className?: string;
};
```

If you want the parent to style this component a `className` should be passed to your `Wrapper` component.